appveyor.bat

```diff
@@ -233,14 +233,17 @@ goto :eof
 @echo on
 cd vim\src
 
+rem Now it's not necessary.  When Vim is compiled, the current patch number
+rem is automatically substituted.
 :: Replace VIM_VERSION_PATCHLEVEL in version.h with the actual patchlevel
-%CYGWIN_DIR%\bin\bash -lc "cd $(cygpath '%APPVEYOR_BUILD_FOLDER%')/vim/src && ../../scripts/patchlevel.sh"
-type version.h
+@rem %CYGWIN_DIR%\bin\bash -lc "cd $(cygpath '%APPVEYOR_BUILD_FOLDER%')/vim/src && ../../scripts/patchlevel.sh"
+@rem call ..\..\scripts\patchlevel.cmd
+@rem type version.h
 
 :: Build GUI version
-nmake -f Make_mvc.mak ^
+nmake.exe -f Make_mvc.mak ^
 	GUI=yes OLE=yes DIRECTX=yes ^
-	FEATURES=HUGE IME=yes MBYTE=yes ICONV=yes DEBUG=no ^
```

Comment:
why do you remove `MBYTE`?

Reply:
This is no longer required, it was used [earlier](https://github.com/vim/vim-win32-installer/pull/336#issuecomment-2011765874). This definition is now always set to "yes".

> MBYTE is an old option which has been removed and always enabled now.

Reply:
I see, thanks